AI/app/data/run/run_faiss.py

```
@@ -0,0 +1,44 @@
+        a.city_code
+    FROM attraction a
+    JOIN attraction_category ac 
+        ON a.attraction_category_id = ac.attraction_category_id
+"""
+
+FAISS_DB_PATH = os.path.join(PROJECT_ROOT, "database/faiss_db")
+
+def main():
+    rows = execute_query(QUERY)
+    attractions, restaurants = convert_to_documents(rows)
+    vector_store_attraction = save_to_faiss(attractions, "관광지", FAISS_DB_PATH+"/attraction")
```

Ruff (0.11.9)

Local variable `vector_store_attraction` is assigned to but never used

Remove assignment to unused variable `vector_store_attraction`

(F841)